sysml.library/Domain Libraries/Geometry/ShapeItems.sysml

```diff
@@ -0,0 +1,249 @@
+package ShapeItems {
+	private import ScalarValues::String;
+	private import ScalarValues::Boolean;
+	private import UnitsAndScales::CoordinateTransformation;
+	private import ISQ::*;
+	private import SI::*;
+	private import Collections::KeyValuePair;
+	private import Collections::Map;
+	private import Objects::*;
+	private import Items::*;
+	private import SequenceFunctions::union;
+	private import CollectionFunctions::contains;
+	private import SequenceFunctions::isEmpty;
```

Comment:
The following imports are unnecessary:
```
private import ScalarValues::String;
private import UnitsAndScales::CoordinateTransformation;
private import Collections::KeyValuePair;
private import Collections::Map;
private import SequenceFunctions::union;
private import CollectionFunctions::contains;
```
The imports from `SI` and `Items` don't need to be `*`:
```
private import SI::m;
private import Items::Item;
```

Reply:
Also fixed locally, awaiting push.

Reply:
Done.